engines/bops_reports/app/views/bops_reports/planning_applications/show.html.erb

```diff
@@ -133,4 +139,39 @@
 
 <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 
+<section id="next-steps">
+  <h3 class="govuk-heading-s"><%= t(".next_steps_heading") %></h3>
+
+  <% if @current_local_authority.submission_guidance_url %>
+    <p class="govuk-body">
+      <%= t(".next_steps_intro_html") %>
+    </p>
+
+    <%= t(".next_steps_list_html") %>
+
+    <p class="govuk-body">
+      <%= t(
+            ".next_steps_outro_html",
+            planning_service_link: @current_local_authority.submission_guidance_url,
+            application_type: @planning_application.recommended_application_type.human_name
+          ) %>
+    </p>
+  <% else %>
+    <p class="govuk-body">
+      You must set the submission guidance url in local authority settings before displaying next steps.
+    </p>
+  <% end %>
+</section>
+
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+<section id="disclaimer">
+  <h3 class="govuk-heading-s"><%= t(".disclaimer_heading") %></h3>
+  <%= govuk_warning_text do %>
+    <%= @planning_application.disclaimer || t(".disclaimer_content") %>
```

Comment:
Would it be worth setting the default disclaimer at the local authority admin level instead? Probably not a big deal but it might not be very clear to an admin that leaving the field blank there will lead to a default being set (that they can't actually see).

Reply:
Yeah fair point but inclined to wait and see if this is a problem